scala-generator/src/main/scala/models/play/files/ModelsGens.scala

```diff
@@ -71,8 +71,8 @@ object ModelsGens {
 	def arbitrary(model: ScalaModel): String = arbitrary(model.ssd.namespaces, model.name, model.qualifiedName)
 	def arbitrary(union: ScalaUnion): String = arbitrary(union.ssd.namespaces, union.name, union.qualifiedName)
   def arbitrary(ns: Namespaces, name: String, tpe: String): String = {
-    val collisionFreeName = s"""${ns.models.split('.').map(_.capitalize).mkString}${name}"""
-    s"""implicit lazy val arbitrary${collisionFreeName}: ${Arbitrary}[$tpe] = ${Arbitrary}(gen${name})"""
+//    val collisionFreeName = s"""${ns.models.split('.').map(_.capitalize).mkString}${name}"""
```

Comment:
@gheine is this comment desired?

Reply:
Yeah, sort of... :)  Couldn't get the genNsStr in Gens.scala to work reliably, scala-check seemed to use it and the default String generator arbitrarily, leading to invalid namespaces being generated. Don't think this is really needed at all though. @plippe any thoughts?